src/migrate/index.js

```diff
@@ -105,9 +107,25 @@ export default class Migrator {
   // dependent on the migration config settings.
   _ensureTable() {
     var table = this.config.tableName;
+    var lockTable = this._getLockTableName(table);
     return this.knex.schema.hasTable(table)
       .then((exists) => {
         if (!exists) return this._createMigrationTable(table);
+      })
+      .then(() => {
+        return this.knex.schema.hasTable(lockTable)
+          .then((exists) => {
```

Comment:
Also... there might be multiple is_locked rows added if multiple processes are racing to run migrations for the first time, but it probably is not a problem, since only the first row is used :)